app/assets/stylesheets/components/_email-signup-section.scss

```diff
@@ -0,0 +1,38 @@
+.app-c-email-signup {
+  @include govuk-responsive-margin(8, "top");
+  padding: govuk-spacing(6);
+  background-color: govuk-colour("light-grey", $legacy: "grey-4");
+  border: 1px solid $govuk-border-colour;
+}
+
+.app-c-email-signup--full-width {
```

Comment:
I'm not sure this accurately describes the behaviour of this option - the component appears be full width by default. Might be better to call this something else - no sides? No borders? Something like that only better 😁